### PR TITLE
Add ROS 2022 to allowable taxonomies for ROS validations

### PR DESCRIPTION
**Pull request by derekgengenbacher-wf**
#### Description:
Added hrefs for ROS 2022 as allowable entry points for ROS validations

**review**:
@hefischer

### CODE REVIEW

**Review by sagesmith-wf — APPROVED** [review]
Multiple keys pointing to the same value seems a bit odd, wondering if it would make more sense to add the year to the RHS or to do a `https://xbrl.frc.org.uk.ireland/FRS-101 in` check for example instead. 🤷

**Review by austinmatherne-wk — APPROVED** [review]

**Comment by hermfischer-wf** [comment]
Agree with Sage's comment on the year in RHS (but need to see how that looks where it is actually used)

**Comment by derekgengenbacher-wf** [comment]
Realistically that does not need to be a dict. It can be a list of allowable taxonomy references. Ill update the commit to reflect that.

File: arelle/plugin/validate/ROS/__init__.py [hunk 1]
```diff
@@ -21,11 +21,14 @@
...
-    "https://xbrl.frc.org.uk/ireland/IFRS/2019-01-01/ie-IFRS-2019-01-01.xsd": "EU IFRS Irish Extension"
-    }
+taxonomyReferences = [
+    "https://xbrl.frc.org.uk/ireland/FRS-101/2019-01-01/ie-FRS-101-2019-01-01.xsd",
+    "https://xbrl.frc.org.uk/ireland/FRS-102/2019-01-01/ie-FRS-102-2019-01-01.xsd",
+    "https://xbrl.frc.org.uk/ireland/IFRS/2019-01-01/ie-IFRS-2019-01-01.xsd",
+    "https://xbrl.frc.org.uk/ireland/FRS-101/2022-01-01/ie-FRS-101-2022-01-01.xsd",
+    "https://xbrl.frc.org.uk/ireland/FRS-102/2022-01-01/ie-FRS-102-2022-01-01.xsd",
+    "https://xbrl.frc.org.uk/ireland/IFRS/2022-01-01/ie-IFRS-2022-01-01.xsd",
+]
```
Comment (austinmatherne-wk) [inline arelle/plugin/validate/ROS/__init__.py:31]: nit: I would keep these sorted

**Review by austinmatherne-wk — APPROVED** [review]

**Review by sagesmith-wf — APPROVED** [review]

File: arelle/plugin/validate/ROS/__init__.py (hunk 1 above)
Comment (sagesmith-wf) [reply arelle/plugin/validate/ROS/__init__.py]: I can go either way on that. I'm imagining a world where the list is long and its actually easier to have it ordered this way unless the datastructure is changed

**Review by sagesmith-wf — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

**Comment by hermfischer-wf** [comment]
approve

**Comment by derekgengenbacher-wf** [comment]
Closing because commit was made here: [master 7aafba2] #130 Add ROS 2022 to allowable taxonomies for ROS

**Pull request closed without merging**